creator-node/src/services/stateMachineManager/stateReconciliation/types.ts

```diff
@@ -1,15 +1,37 @@
 // Issue sync request job
+
+export type SyncRequestAxiosData = {
+  wallet: string[]
+  creator_node_endpoint?: string
+  sync_type?: string
+  immediate?: boolean
+  forceResync?: boolean
+  timestamp?: string
+  signature?: string
+}
+
 export type SyncRequestAxiosParams = {
   baseURL: string
   url: string
   method: string
-  data: {
-    wallet: string[]
-    creator_node_endpoint?: string
-    sync_type?: string
-    immediate?: boolean
-  }
+  data: SyncRequestAxiosData
 }
+
+export type ForceResyncAuthParams = {
+  data: SyncRequestAxiosParams
+  timestamp: string
+  signature: string
```

Comment:
I'm a little confused why this needs a separate timestamp and signature if the `data: SyncRequestAxiosParams` property already has a timestamp and signature?

Reply:
yeah I realized I set up the typing incorrectly 😓 pushing a fix!